.devcontainer/Dockerfile

```diff
@@ -1,120 +1,4 @@
-# [Choice] bionic (18.04), focal (20.04)
-ARG VARIANT="focal"
-FROM ubuntu:${VARIANT}
+# FROM ghcr.io/cpp_best_practices/cpp:0.1.0   # TODO: activate
+FROM ghcr.io/ddalcino/cpp:0.1.0
```

Comment:
How the variables like ${VARIANT}, ${LLVM_VER} etc. are passed from the devcontainer.json file to the actual dev container that VSCode use with setup?

Reply:
We can add a matrix to `.github/workflows/publish-dockerfile.yml` that builds separate containers with all the different variables, and names them appropriately. It should not be too difficult to do.